docs/getstarted/evals.md

```diff
@@ -0,0 +1,221 @@
+    }
+]
+```
+
+```python
+from datasets import load_dataset
+from ragas import EvaluationDataset
+eval_dataset = load_dataset("explodinggradients/earning_report_summary",split="train")
+eval_dataset = EvaluationDataset.from_hf_dataset(eval_dataset)
+print("Features in dataset:", eval_dataset.features())
+print("Total samples in dataset:", len(eval_dataset))
+```
+
+Output
+```
+Features in dataset: ['user_input', 'response']
+Total samples in dataset: 50
+```
+
+Evaluate using dataset
+
+```python
+results = evaluate(eval_dataset, metrics=[metric])
+results
+```
+
+Output
+```
+{'summary_accuracy': 0.84}
+```
+
+This score shows that out of all the samples in our test data, only 84% of summaries passes the given evaluation criteria.
+
+Export the sample level scores to pandas dataframe
+
+```python
+results.to_pandas()
+```
+
+Output
+```
+	user_input	                                        response	                                        summary_accuracy
+0	summarise given text\nThe Q2 earnings report r...	The Q2 earnings report showed a 15% revenue in...	1
+1	summarise given text\nIn 2023, North American ...	Companies are strategizing to adapt to market ...	1
+2	summarise given text\nIn 2022, European expans...	Many companies experienced a notable 15% growt...	1
+3	summarise given text\nSupply chain challenges ...	Supply chain challenges in North America, caus...	1
+```
+
+Viewing the sample-level results in a CSV file, as shown above, is fine for quick checks but not ideal for detailed analysis or comparing results across evaluation runs. For a better experience, use [app.ragas.io](https://app.ragas.io/) to view, analyze, and compare evaluation results interactively.
+
+
+## Analyzing Results
+
+For this you may sign up and setup [app.ragas.io]() easily. If not, you may use any alternative tools available to you. 
+
+In order to use the [app.ragas.io](http://app.ragas.io) dashboard, you need to have an account on [app.ragas.io](https://app.ragas.io/). If you don't have one, you can sign up for one [here](https://app.ragas.io/login). You will also need to generate a [Ragas API key](https://app.ragas.io/dashboard/settings/app-tokens).
+
+Once you have the API key, you can use the `upload()` method to export the results to the dashboard.
+
+```python
+import os
+os.environ["RAGAS_API_KEY"] = "your_api_key"
+```
+
+Now you can view the results in the dashboard by following the link in the output of the `upload()` method.
+
+```python
+results.upload()
+```
+
+![](../_static/imgs/ragas_get_started_evals.gif)
+
+
+
+## Aligning Metrics
+
+In the example above, we can see that the LLM-based metric mistakenly marks some summary as accurate, even though it missed critical details like growth numbers and market domain. Such mistakes can occur when the metric does not align with your specific evaluation preferences. For example, 
+
+![](../_static/imgs/eval_mistake1.png)
```

Comment:
in this section we have not mentioned editing the reason, isn't that important?

Reply:
an image of editing the reason would have been very helpful I think

Reply:
I have left a link for how to train guide, which should handle it. Here those details may be a little too detailed.